lib/kitchen/driver/openstack.rb

```diff
@@ -101,6 +101,23 @@ def destroy(state)
         state.delete(:hostname)
       end
 
+      def env_cmd(cmd)
```

Comment:
Won't this break HTTP proxying since that's handled in the [parent class' env_cmd](https://github.com/test-kitchen/test-kitchen/blob/master/lib/kitchen/driver/ssh_base.rb#L133)? It also doesn't seem specific to OpenStack. Would this be better as a new feature in Test Kitchen so other drivers could use it too?